src/components/EmojiPicker/EmojiPickerMenu/index.native.js

```diff
@@ -126,34 +78,35 @@ function EmojiPickerMenu({preferredLocale, onEmojiSelected, preferredSkinTone, t
      * @param {Object} item
      * @returns {*}
      */
-    const renderItem = ({item}) => {
-        const {code, types} = item;
-        if (item.spacer) {
-            return null;
-        }
+    const renderItem = useCallback(
+        ({item, target}) => {
+            const {code, types} = item;
+            if (item.spacer) {
+                return null;
+            }
+
+            if (item.header) {
+                return (
+                    <View style={[styles.emojiHeaderContainer, target === 'StickyHeader' ? styles.mh4 : {width: windowWidth}]}>
```

Comment:
Coming from https://github.com/Expensify/App/issues/33709#issuecomment-1871727444

We had a bug here scroll not working over header.
RCA: emoji header is wrapped in an absolute container and overlaps with scroll bar